host/src/main.rs

```diff
@@ -36,26 +39,16 @@ pub struct Opt {
     /// [default: 0.0.0.0:8080]
     address: String,
 
-    #[structopt(long, require_equals = true, default_value = "/tmp")]
+    #[structopt(long, require_equals = true, default_value = "/tmp/raiko-host")]
```

Comment:
Default should be no caching I think because of potential problems with caching:
- Input data could be outdated (especially when developing, but also when people update raiko in the future)
- when running with caching the cache folder will keep growing and growing and take up a lot of space